tests/lint-roller-markdown-api-history.spec.ts

```diff
@@ -0,0 +1,431 @@
+import { spawnSync } from 'node:child_process';
+import path, { resolve } from 'node:path';
+import { readdir, unlink, writeFile } from 'node:fs/promises';
+
+import { afterAll, beforeAll, describe, expect, it } from 'vitest';
+
+const FIXTURES_DIR = resolve(__dirname, 'fixtures');
+const MOCKUP_API_HISTORY_SCHEMA = resolve(FIXTURES_DIR, 'mockup-api-history.schema.json');
+const MOCKUP_BREAKING_CHANGES_FILE = resolve(FIXTURES_DIR, 'mockup-breaking-changes.md');
```

Comment:
I'd prefer we drop the `mockup-` prefix and fill the contents of these files with the latest from their respective files. It ensures we're testing against realistic content for them, and if we hit a case on `e/e` which is valid but handled incorrectly by this code, we can simply drop that new content into these files in the PR that fixes the issue and be confident it's fixed.

Reply:
Resolved in [`bbdc778` (#73)](https://github.com/electron/lint-roller/pull/73/commits/bbdc77801728321e7476fcfd314e4ddc575e9b85) and [`441c8a0` (#73)](https://github.com/electron/lint-roller/pull/73/commits/441c8a0978c1621f316fc999a214e43fcb78f6f2)